performance-tests/pom.xml

```diff
@@ -104,6 +104,12 @@
         <plugins>
 
             <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.plugin.version}</version>
+            </plugin>
```

Comment:
Ok that is easy enough to fix then. I will have to override the surefire plugin config for these two modules to use argLine.